src/upload/system/library/transbank/composer.json

```
@@ -1,6 +1,14 @@
 {
     "require": {
-        "transbank/transbank-sdk": "~2.0"
+        "transbank/transbank-sdk": "~2.0",
+        "monolog/monolog": "^1.27"
+    },
+    "autoload": {
+        "psr-4": {
+            "Transbank\\SDK\\": "system/library/Transbank/SDK/",
+            "Transbank\\Utils\\": "system/library/Transbank/utils/",
+            "Transbank\\Exceptions\\": "system/library/Transbank/Exceptions/"
```

This is not necessary, if the base path is the same

```suggestion
        "psr-4": {
            "Transbank\\Opencart\\Webpay\\": "system/library/Transbank/",
```